db/migrations/20180320162959_users.js

```diff
@@ -9,7 +9,9 @@ exports.up = async (knex) => {
     table.text('last_name').notNullable();
     table.text('email').notNullable();
     table.text('role').notNullable();
-    table.boolean('registration_completed').defaultTo(false);
+
+    // Status of the account 'COMPLETE_REGISTRATON', 'BLOCKED', 'REGISTERD'
+    table.text('status').notNullable().defaultTo('COMPLETE_REGISTRATON')
 
```

Comment:
The `status` should be a FK to a `CODE`, which belongs to a codetype `USER_STATUSES`